pkg/api/types.go

```diff
@@ -59,14 +59,9 @@ type Request struct {
 	// WorkingDir describes temporary directory used for downloading sources, scripts and tar operations.
 	WorkingDir string
 
-	// LayeredBuild describes if this is build which layered scripts and sources on top of BaseImage.
+	// LayeredBuild describes if this is build which layered scripts and sources on top of BuilderImage.
 	LayeredBuild bool
 
-	// InstallDestination allows to override the default destination of the STI
-	// scripts. It allows to place the scripts into application root directory
-	// (see ONBUILD strategy). The default value is "upload/scripts".
-	InstallDestination string
```

Comment:
pkg/build/strategies/sti/sti.go
71:	inst := script.NewInstaller(req.BaseImage, req.ScriptsURL, req.InstallDestination, docker)

This was originally used here, for some reason it is not anymore. Is there a reason why?